lnst/Recipes/ENRT/MPTCPRecipe.py

```diff
@@ -71,23 +71,29 @@ def test_wide_configuration(self):
             host.run("sysctl -w /net/mptcp/enabled=1")
             host.eth0.ip_add(ipaddress("192.168.101." + str(i+1) + "/24"))
             host.eth1.ip_add(ipaddress("192.168.102." + str(i+1) + "/24"))
-            #TODO ASK MPTCP DEVs whats the deal with v6? Can we test it at the same time as v4 or does it have to be a seperate run.
-            #host.eth0.ip_add(ipaddress("fc00::" + str(i+1) + "/64"))
-            #host.eth1.ip_add(ipaddress("fc01::" + str(i+1) + "/64"))
+            host.eth0.ip_add(ipaddress("fc00::" + str(i+1) + "/64"))
+            host.eth1.ip_add(ipaddress("fc01::" + str(i+1) + "/64"))
             host.eth0.up()
             host.eth1.up()
             config.test_wide_devices.append(host.eth0)
             config.test_wide_devices.append(host.eth1)
-            config.mptcp_endpoints.append(host.eth1)
 
-        #Configure endpoints
-        #TODO Might need to redo this
-        # ASK MPTCP DEV we  might not need to configure endpoints on both sides,
-        # or if we do the optioons might be different (ie subflows only on client side)
-        for ep_dev in config.mptcp_endpoints:
-            #TODO check with MPTCP devs about ipv6 support
-            ep_dev.netns.mptcp.add_endpoints(ep_dev.ips_filter(family=AF_INET))
-            #ep_dev.netns.mptcp.add_endpoints(ep_dev.ips_filter(family=AF_INET6))
+        # Configure endpoints only host1.eth1
+        if "ipv4" in self.params.ip_versions:
+            host1.mptcp.add_endpoints(host1.eth1.ips_filter(family=AF_INET), flags=MPTCPFlags.MPTCP_PM_ADDR_FLAG_SUBFLOW)
+            # Need route on client side to populate forwarding table
+            host1.run(f"ip route add 192.168.101.0/24 dev {host1.eth1.name} via 192.168.102.2 prio 10000")
+            # Need to disable rp_filter on server side
+            host2.run("sysctl -w net.ipv4.conf.all.rp_filter=0")
```

Comment:
We need to make sure the sysctl variables are restored once the recipe completes.